client/my-sites/subscribers/main.tsx

```diff
@@ -177,6 +180,23 @@ const SubscribersPage = ( {
 		setGiftUsername( display_name );
 	};
 
+	const Gate = ( props: { children: ReactNode } ) => {
```

Comment:
I never thought of defining a component inside another component, but it makes perfect sense here 👍